### PR TITLE
Cherry-pick to 6.7: Add support in jolokia get response for single value (#11075)

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
Original message:
Jolokia module fails to parse some responses fetched using the GET method with error:
```
json: cannot unmarshal number into Go struct field Entry.Value of type map[string]interface {}
failed to unmarshal jolokia JSON response '{"request":{"mbean":"java.lang:type=Runtime","attribute":"Uptime","type":"read"},"value":88622,"timestamp":1551739190,"status":200}'
```
This is because value field can be a map[string]interface or it can also be just a float.

(cherry picked from commit c238ea27ffdc1c5614dcef924faf7df55b4c5700)

### CODE REVIEW

File: metricbeat/module/jolokia/jmx/_meta/data.json [hunk 1]
```diff
@@ -1,33 +1,34 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
+    "event": {
+        "dataset": "jolokia.testnamespace",
+        "duration": 115000,
+        "module": "jolokia"
```
Comment (jsoriano) [inline metricbeat/module/jolokia/jmx/_meta/data.json]: I think this is not expected to change in 6.X, could you regenerate the data.json with this version?

Comment (kaiyan-sheng) [reply metricbeat/module/jolokia/jmx/_meta/data.json]: Yep :-) Will do!

Comment (jsoriano) [reply metricbeat/module/jolokia/jmx/_meta/data.json]: Or leave it as it was in master.

File: metricbeat/module/jolokia/jmx/_meta/data.json [hunk 2]
```diff
@@ -4,30 +4,34 @@
...
-            "uptime": 580487
+            "uptime": 1538717
         }
     },
     "metricset": {
-        "host": "jolokia:8778",
+        "host": "127.0.0.1:8080",
         "module": "jolokia",
         "name": "jmx",
-        "namespace": "testnamespace",
```
Comment (jsoriano) [inline metricbeat/module/jolokia/jmx/_meta/data.json]: Umm, this is changing :thinking: 

Though I don't think this is related with this change.

Comment (kaiyan-sheng) [reply metricbeat/module/jolokia/jmx/_meta/data.json]: Yeah I regenerated `data.json` without this backport pr and namespace got changed as well hmm

Comment (ruflin) [reply metricbeat/module/jolokia/jmx/_meta/data.json]: This is a change we did quite some time ago. Seems like we never regenerated the file since then 🤔

Comment (kaiyan-sheng) [reply metricbeat/module/jolokia/jmx/_meta/data.json]: good to know! thanks!! Next time we will make sure to regenerate `data.json` 😄

**Pull request merged**